Build/Build.cs

```diff
@@ -263,10 +263,10 @@ public override void ConfigureTestMatrix (TestMatricesBuilder builder)
         "WebTestingTestMatrix",
         new TestDimension[,] // todo docker images need to be wired to the config file
         {
-            { Chrome, NET48, Debug, x86, NoDB, EnforcedLocalMachine(Docker_Win_NET48) },
-            { Chrome, NET481, Release, x86, NoDB, EnforcedLocalMachine(Docker_Win_NET481) },
-            { Firefox, NET48, Release, x64, NoDB, EnforcedLocalMachine(Docker_Win_NET481) },
-            { Edge, NET48, Release, x64, NoDB, EnforcedLocalMachine(Docker_Win_NET48) },
+            { Chrome, NET8_0, Debug, x86, NoDB, EnforcedLocalMachine(Docker_Win_NET8_0) },
+            { Chrome, NET8_0, Release, x86, NoDB, EnforcedLocalMachine(Docker_Win_NET8_0) },
+            { Firefox, NET8_0, Release, x64, NoDB, EnforcedLocalMachine(Docker_Win_NET8_0) },
+            { Edge, NET8_0, Release, x64, NoDB, EnforcedLocalMachine(Docker_Win_NET8_0) },
```

Comment:
Using .NET here as target might not work if the test projects target `net8.0-windows` instead. Until we do the .NET support we should be able to keep using `NET48`/`NET481` instead of `NET8_0`

Reply:
Sure. Changed it because we have a test which checks for the same platform for server and instance, it seems the server is always run in 64bit. Do we still need to configure something here? Why do we even support 32 bit anymore?

Reply:
Changing to NET8_0 fixed this